src/Diagnostics.RuntimeHost/Services/DevOpsClient/DevOpsClient.cs

```diff
@@ -112,6 +116,13 @@ public async Task<object> GetFileContentAsync(string filePathInRepo, string reso
 
             try
             {
+                GitRepository repositoryAsync = await gitClient.GetRepositoryAsync(_project, _repoID, (object)null, new CancellationToken());
```

Comment:
I think we should use the API cancellation token which has the 60 sec timeout instead of creating new here.

Reply:
This API call was actually not needed (ref: Darrel's comment on the same file), so I've removed it. For other such calls, I will add the time out.